x/curating/types/params.go

```diff
@@ -73,7 +73,7 @@ func NewParams(
 
 // String implements the stringer interface for Params
 func (p Params) String() string {
-	out, _ := yaml.Marshal(p)
+	out, _ := yaml.Marshal(p) // nolint
```

Comment:
What was the linting issue here I don't think we should ignore the issue , maybe check it and return an empty string instead?

Reply:
Refactored this.